crypto/bls/private.go

```diff
@@ -0,0 +1,52 @@
+// Copyright (C) 2023, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package bls
+
+import (
+	"errors"
+
+	blst "github.com/supranational/blst/bindings/go"
+
+	"github.com/ava-labs/avalanchego/utils/crypto/bls"
+)
+
+const PrivateKeyLen = blst.BLST_SCALAR_BYTES
```

Comment:
recommend changing this to `const PrivateKeyLen = bls.SecretKeyLen` so we don't need to keep our reference to `blst` (I think there are a few places you do something like this)